species/fit/retrieval.py

```diff
@@ -1056,6 +1068,17 @@ def run_multinest(
         else:
             self.abund_nodes = None
 
+        # set default gradient priors if applicable
```

Comment:
Could you run [black](https://black.readthedocs.io) on this file? It will automatically fix some style things.

Reply:
I did this via a web app in commit [7be8239](https://github.com/tomasstolker/species/pull/82/commits/7be8239fcefc3e54103de0656dd4a2638e1300f8), hopefully that worked?

Reply:
Thanks, yes looks good!